### PR TITLE
ML 2.19 : Gradient Descent Algorithms (A)

### PR DESCRIPTION
**Pull request by ron352**
<hr>

## Description 📜
Include a brief description of  gradient descent, real life example , lost function etc.
 
Fixes # <2950> 

<hr>

## Type of change 📝

- [x] Au#dio (Should be in mp3 format Includes speech clarity, Concise ,Low distortion)
 

<hr>

## Domain of Contribution 📊

- [x] Machine #Learning

<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

## Screenshots / Gif (Optional) 📸

<hr>

### CODE REVIEW

**Comment by ron352** [comment]
Audio Content :

Loss Function
Gradient Descent algorithm
Real life example
Applications
Disadvantages

@abhisheks008 👍

**Comment by abhisheks008** [comment]
Nothing to say about your PRs. Excellent work done 👏. PR Approved 🎉
